src/surfaces/actionBar/actionBarActionsButtonDrawerSubOption.jsx

```diff
@@ -97,22 +94,27 @@ class ActionBarActionsButtonDrawerSubOption extends React.PureComponent {
             subOption,
             subOptionClassNameNum,
         } = this.props;
-        const isSubOptionDisabled = subOption.disable || subOption.disabled;
+
+        const isSubOptionDisabled = subOption.disable;
         const classNameNumber = subOptionClassNameNum;
+        const subOptionClassName = 'actions_button_drawer--sub_option';
+
         const subOptionClasses = ClassNames(
-            'actions_button_drawer--sub_option',
-            `actions_button_drawer--sub_option-${classNameNumber}`,
-            `${isSelected ? `actions_button_drawer--sub_option-${classNameNumber}-show` : ''}`,
-            `${isSubOptionDisabled ? `actions_button_drawer--sub_option-${classNameNumber}-disabled` : ''}`,
+            subOptionClassName,
+            `${subOptionClassName}-${classNameNumber}`,
+            {
+                [`${subOptionClassName}-${classNameNumber}-show`]: isSelected,
+                [`${subOptionClassName}-${classNameNumber}-disabled`]: isSubOptionDisabled,
+            },
         );
 
-        /* eslint-disable jsx-a11y/click-events-have-key-events */
-        /* eslint-disable jsx-a11y/interactive-supports-focus */
         return (
             <div
                 className={subOptionClasses}
                 onClick={this.onClick}
+                onKeyDown={this.onKeyDown}
                 role="menuitem"
+                tabIndex={0}
```

Comment:
Just confirming: `tabIndex` and `onKeyDown` completely appease the `jsx-a11y` ESLINT warnings?

Reply:
Yes, and also it's helpful to have `0` as the value so we can actually tab these actions and also hit enter/return. I am trying to focus on some keyboard things in 10.0.0.... trying.

Reply:
Copy that.  Carry on, good sir!
Resolving.